### ISSUE TITLE
Tree Edit Distance (TED) based accuracy score: 0.0, F1 accuracy score: 0.0 on test data

### ISSUE THREAD

**Issue by Sherelle20** [clovaai/donut#332]
hello please need help, I train a donut classification model on 40 epochs when testing I got this output

Resolving data files: 100%|██████████████| 1061/1061 [00:00<00:00, 20971.22it/s]
Resolving data files: 100%|████████████████| 358/358 [00:00<00:00, 24851.64it/s]
Resolving data files: 100%|████████████████| 359/359 [00:00<00:00, 23192.94it/s]
0%| | 0/358 [00:00<?, ?it/s]/usr/local/lib/python3.11/site-packages/transformers/generation/configuration_utils.py:453: UserWarning: num_beams is set to 1. However, early_stopping is set to True -- this flag is only used in beam-based generation modes. You should set num_beams>1 or unset early_stopping.
warnings.warn(
100%|█████████████████████████████████████████| 358/358 [04:44<00:00, 1.26it/s]
Total number of samples: 358, Tree Edit Distance (TED) based accuracy score: 0.0, F1 accuracy score: 0.0

Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Some weights of DonutModel were not initialized from the model checkpoint at /kaggle/working/result_100/train_docvqa/test_experiment and are newly initialized: ['decoder.model.lm_head.weight']

**Comment by Tottenham37** [comment]
Did you find the solution?
